katas/content/single_qubit_gates/phase_change/solution.md

```diff
@@ -0,0 +1,38 @@
+We know that:
+
+$$
+R1(\alpha)=
+ \begin{bmatrix}1 & 0\\\0 & \color{red}{e^{i\alpha}}\end{bmatrix}
+$$
+
+So we have:
+
+$$
+R1(\beta |0\rangle + \gamma |1\rangle) =
+ \begin{bmatrix}1 & 0 \\\0 & \color{red}{e^{i\alpha}} \end{bmatrix}
+ \begin{bmatrix}\beta\\\ \gamma\\\ \end{bmatrix}=
+\begin{bmatrix}1.\beta + 0.\gamma\\\ 0.\beta + {\color{red}{e^{i\alpha}}}\gamma\end{bmatrix}=
+ \begin{bmatrix}\beta\\\ {\color{red}{e^{i\alpha}}}\gamma\end{bmatrix}=
+ \beta |0\rangle + {\color{red}{e^{i\alpha}}} \gamma |1\rangle
+$$
+
+> Note that the results produced by the test harness can be unexpected.
```

Comment:
This is something we're working on in #1317, and the numbers in the note are based on the results produced by Classic QDK, which differ from the ones we have in the Modern QDK. I think let's remove the note for now, and then I'll revisit it after we get the global phase issue in reasonable shape.